File: unit_2/w05d04/morning_exercise/stack.js
"use strict";

var stack = {
  // Write your code here
  myStack: [],
  add: function(thing){
    this.myStack.push(thing);
    return thing;
  },
  peek: function(){
    return this.myStack[this.myStack.length-1];
  },
  remove: function(){
    return this.myStack.pop();
  }
}

module.exports = stack;

File: unit_4/w11d01/morning_exercise/stack.rb
class Stack
  def initialize
    @stack = []
  end
  def new
    return @stack
  end
  def push(el)
    @stack.push(el)
    return el
  end
  def pop
    @stack.pop
  end
  def peek
    return @stack[@stack.length-1]
  end
end
